### DISCUSSION TITLE
Chrome extension on Opera

### DISCUSSION THREAD

**Discussion by Rikk** [Tampermonkey/tampermonkey#1637] [Q&A] [answered]
Does the extension from Chrome Store currently works the same on Opera browser? Or, does the [Opera version](https://addons.opera.com/en/extensions/details/tampermonkey-beta/) includes some kind of compatibility specially for Opera?

**Comment by derjanb** [comment, marked as answer]
The Opera version is reviewed by the Opera Store team, which often takes some time. That's why this version is updated less often. Besides this there is no difference to the Chrome version.

**Reply by Rikk** [reply]
Thanks @derjanb. I found they are blocking from running the version from Chrome Store, actually.

**Comment by Rikk** [comment]
Will addon for Opera get updated? 4.18 didn't got in.
https://addons.opera.com/en/extensions/details/tampermonkey-beta/

**Comment by derjanb** [comment]
> Status has changed for Tampermonkey
> Tampermonkey v4.19.6183 on Opera Developer: Published.
> Tampermonkey v4.19.6183 on Opera Next: Published.
> 
> Moderator's note:
> 
> Your extension needs to be manually reviewed and moderated before it can be added to Opera Stable. It is currently only viewable and available for download in Opera Developer and Opera Beta.

I've updated the Opera version. Thanks for the hint. I really should automate release uploads... 😃

**Reply by Rikk** [reply]
![image](https://github.com/Tampermonkey/tampermonkey/assets/2916485/2cb3be84-9e33-4058-ade5-c5d5ff09a4e4)
Let's hope they publish it this time, more than a year without update...

Btw, on another topic, can Tam be re-enabled after being disabled? I don't see it in settings.

**Reply by derjanb** [reply]
> Let's hope they publish it this time, more than a year without update...

It usually took some time in the past.

> Btw, on another topic, can Tam be re-enabled after being disabled? I don't see it in settings.

At the moment not via a setting, but you can run this at the console of any extension page (like the options page).

```
localStorage.removeItem('monkey_clip')
```
